data/virtualization/autoyast/guest_12.xml.ep

```diff
@@ -0,0 +1,334 @@
+    <global>
+      <activate>true</activate>
+    % if ($vm_name !~ /PV/) {
+      <append>console=ttyS0,115200 console=tty0</append>
+    % }
+      <boot_root>true</boot_root>
+      <gfxmode>auto</gfxmode>
+      <hiddenmenu>false</hiddenmenu>
+      <os_prober>false</os_prober>
+      <terminal>console</terminal>
+      <timeout config:type="integer">8</timeout>
+      <trusted_grub>false</trusted_grub>
+    </global>
+    <loader_type>grub2</loader_type>
+  </bootloader>
+  <general>
+    <ask-list config:type="list"/>
+    <mode>
+      <confirm config:type="boolean">false</confirm>
+      <final_reboot config:type="boolean">true</final_reboot>
+    </mode>
+    <proposals config:type="list"/>
+    <signature-handling>
+      <accept_file_without_checksum config:type="boolean">true</accept_file_without_checksum>
+      <accept_non_trusted_gpg_key config:type="boolean">true</accept_non_trusted_gpg_key>
+      <accept_unknown_gpg_key config:type="boolean">true</accept_unknown_gpg_key>
+      <accept_unsigned_file config:type="boolean">true</accept_unsigned_file>
+      <accept_verification_failed config:type="boolean">true</accept_verification_failed>
+      <import_gpg_key config:type="boolean">true</import_gpg_key>
+    </signature-handling>
+    <storage>
+      <partition_alignment config:type="symbol">align_optimal</partition_alignment>
+      <start_multipath config:type="boolean">false</start_multipath>
+    </storage>
+  </general>
+  <suse_register>
+  <do_registration config:type="boolean">true</do_registration>
+    <email/>
+    <reg_code>{{SCC_REGCODE}}</reg_code>
+    <install_updates config:type="boolean">true</install_updates>
+    <slp_discovery config:type="boolean">false</slp_discovery>
+    <!--  optionally register some add-ons   -->
+    <addons config:type="list">
+    <addon>
+    <name>sle-module-adv-systems-management</name>
+    <version>12</version>
+    <arch>{{ARCH}}</arch>
+    <release_type/>
+    </addon>
+    % if ($ltss_code) {
+    <addon>
+      <name>SLES-LTSS</name>
+      <version>{{VERSION}}</version>
+      <arch>{{ARCH}}</arch>
+     <reg_code><%= $ltss_code %></reg_code>
+     <release_type/>
+    </addon>
+    % }
+    </addons>
+  </suse_register>
+  <add-on>
+    <add_on_products config:type="list">
+      <listentry>
+        <media_url>http://download.suse.de/ibs/SUSE:/CA/{{CA_STR}}/</media_url>
+      </listentry>
+      % for my $repo (@$repos) {
+      <listentry>
+        <media_url><%= $repo %></media_url>
+      </listentry>
+      % }
+    </add_on_products>
+  </add-on>
+  <firewall>
+    <enable_firewall config:type="boolean">false</enable_firewall>
+    <start_firewall config:type="boolean">false</start_firewall>
+  </firewall>
+  <keyboard>
+    <keymap>english-us</keymap>
+  </keyboard>
+  <language>
+    <language>en_US</language>
+    <languages/>
+  </language>
+  <login_settings/>
+  <networking>
+    <dhcp_options>
+    <dhclient_client_id><%= $vm_name %></dhclient_client_id>
```

Comment:
It will be better to pay attention to indent overall, having two+ blanks under its parent item.

Reply:
I do not think it is prettify. It is not well readable without proper indent -- the same requirement as BASIC coding rule in PRs, each line should start with proper blanks.